espresso/view-store/view_store_binary_tree.go

```diff
@@ -0,0 +1,97 @@
+package view_store
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type View struct {
+	viewNumber        uint64
+	builderCommitment string
```

Comment:
Why it is called `builderCommitment`? Is `builder` necessary?

Reply:
Builder is using this instead of payload commitment so I decided to use it as well: https://espressosys.slack.com/archives/C08RH5EQ66A/p1755275600175529

Doesnt really matter, we just need some unique identification